guacamole-common/src/main/java/org/apache/guacamole/protocol/ConfiguredGuacamoleSocket.java

```diff
@@ -63,12 +64,39 @@
      */
     private GuacamoleProtocolVersion protocolVersion =
             GuacamoleProtocolVersion.VERSION_1_0_0;
-    
+
+    /**
+     * Parses the arguments for the Guacamole "error" server command and returns
+     * the corresponding exception.
+     * @param args The arguments as provided by the server command.
+     * @return An instance of {@link GuacamoleServerErrorCommandException} configured
+     *         with the server-provided arguments, or {@literal null} if the specified
+     *         arguments are invalid.
+     */
+    private static GuacamoleServerErrorCommandException parseServerErrorCommandArgs(List<String> args) {
+        if (args == null || args.size() != 2)
+            return null;
```

Comment:
1. It's pretty common for instructions to be extended down the line by adding new arguments, relying on older implementations to ignore any additional arguments. I think it would be better here to test `args.size() < 2`.
2. Under what circumstance would `args` be `null`?

Reply:
1. Fixed
2. None, apparently. I've changed the signature per your other comments, so this is no longer an issue.